hydroflow/tests/surface_codegen.rs

```diff
@@ -234,6 +234,39 @@ pub fn test_anti_join() {
     assert_eq!(&[(1, 2), (2, 3), (3, 4), (4, 5), (5, 4), (6, 5)], &*out);
 }
 
+#[multiplatform_test]
+pub fn test_buffer() {
+    let (buffer1_tx, buffer1_rx) = hydroflow::util::unbounded_channel::<()>();
+    let (buffer2_tx, buffer2_rx) = hydroflow::util::unbounded_channel::<()>();
+    let (tx, mut rx) = hydroflow::util::unbounded_channel::<()>();
+    let mut df = hydroflow_syntax! {
+        my_tee = tee();
```

Comment:
Is there a better way of testing push/pull than this? It is kind of indirect and could silently break if some push/pull optimizations are implemented that change the behavior of this code gen.

Reply:
Right now no... but it could be added as a debugging feature somehow, wouldn't be hard to modify the subgraph partitioning algorithm